### DISCUSSION TITLE
Usage of the Raspberry Pi camera in "bullseye"

### DISCUSSION THREAD

**Discussion by chrstnkglr** [bluenviron/mediamtx#743] [Questions]
I have tried to connect the Raspi camera without success. Unfortunately, the following message keeps coming up: no element "rpicamsrc" 
Could it be that it is because of libcamera which is the new standard at Raspberry Pi OS Bullseye?
Do you already have experience how I could solve this? Thanks for the advice :)

**Comment by aler9** [comment]
use `libcamerasrc` as described here:

https://qengineering.eu/install-gstreamer-1.18-on-raspberry-pi-4.html

**Comment by renarena** [comment]
Unfortunately this doesnt help me in constructing a proper path. I got errors and i dont know all the syntax i maybe have to adapt.
I think a working line as example would help other people too.

This works 
`gst-launch-1.0 libcamerasrc ! video/x-raw, width=640, height=480, framerate=30/1 ! videoconvert ! videoscale ! autovideosink` 
but this not 
`gst-launch-1.0 libcamerasrc ! video/x-raw, width=640, height=480, framerate=30/1 ! videoconvert ! videoscale ! rtspclientsink location=rtsp://localhost:8554/hqstream debug=false`

[a comment by an automated account is not shown]
